apps/docs/utils/getSectionLinks.ts

```diff
@@ -0,0 +1,23 @@
+import type { MDX } from "contentlayer/core";
+
+function getSectionLinks (content: {
+    body: MDX;
+}) {
+    const regex = /##\s*([\w\s]+)/g;
```

Comment:
This regex is faulty in some situations, this should be :  `/(?<=##\s).*?(?=\n)/g` 
I fixed this in my current branch.